src/python/pants/backend/helm/util_rules/chart.py

```diff
@@ -330,12 +425,27 @@ async def get_helm_chart(request: HelmChartRequest) -> HelmChart:
         ),
     )
 
-    # Merge all sources together
-    all_sources = await Get(
+    # Merge chart content
```

Comment:
Nit: comments should end with periods. See https://www.pantsbuild.org/docs/style-guide ... not a blocker, but something to do moving forward.